cedar-policy-validator/src/typecheck/test_optional_attributes.rs

```diff
@@ -776,30 +778,17 @@ fn action_attrs_failing() {
         )],
     );
 
-    // Doesn't fail do to imprecision in ActionEntity LUB computation requiring `may_have_attr` to return true for ActionEntity types
+    // Doesn't fail due to imprecision in ActionEntity LUB computation requiring `may_have_attr` to return true for ActionEntity types
```

Comment:
Despite the comment, this actually does fail, right? if "fail" means that validation fails, rather than that the test fails

Reply:
Yes. We changed this so that we know the attributes of action entities precisely (there typically being none, but code exists to handle attribute if they exist).